bindings/rascal/representations/coulomb_matrix.py

```diff
@@ -38,8 +39,7 @@ class SortedCoulombMatrix(object):
         Physical Review Letters, 108(5), 58301. https://doi.org/10.1103/PhysRevLett.108.058301
     """
 
-    def __init__(self, cutoff, sorting_algorithm='row_norm', size=10, central_decay=-1, interaction_cutoff=10, interaction_decay=-1,
-                 method='thread', n_workers=1, disable_pbar=False):
+    def __init__(self, cutoff, sorting_algorithm='row_norm', size=10, central_decay=-1, interaction_cutoff=10, interaction_decay=-1):
```

Comment:
Crazy long line here (the documentation should also be wrapped to 72 characters, see PEP8)

Reply:
autopep8 does not work very well with [line breaks](https://github.com/hhatto/autopep8/issues/393) (one of the many example I found after a quick search). So the tool does not work at the moment. Should we ga back to format by hand ?

Reply:
Hmm, perhaps, as long as autopep8 doesn't mess with it again after hand-formatting